src/auth/oauth2client.ts

```diff
@@ -660,20 +674,28 @@ export class OAuth2Client extends AuthClient {
     options: GetTokenOptions
   ): Promise<GetTokenResponse> {
     const url = this.endpoints.oauth2TokenUrl.toString();
-    const values = {
+    const headers: any = {'Content-Type': 'application/x-www-form-urlencoded'};
```

Comment:
Don't need to use any here, can do something like this instead, same with below

https://github.com/googleapis/google-auth-library-nodejs/blob/7dbedff829be093dad0a2da32f0ab12862d473bc/src/auth/awsrequestsigner.ts#L142C5-L155C6

Reply:
Thanks for the example. When I made this change, I tried something else and none of them pass the generic type check then I left an `any`. Changed upon it.